src/signature.cpp

```diff
@@ -23,6 +23,10 @@ bool verify_rsa(const char* publicKey, size_t pubKeyLen, const char* message,
 // static
 bool Signature::verify(const QByteArray& publicKey, const QByteArray& content,
                        const QByteArray& signature) {
+  if (publicKey == nullptr || content == nullptr || signature == nullptr) {
+    return false;
+  }
```

Comment:
Not too sure about the consequences of this change, I needed something that would make signature verification fail in testing mode. This seemed logical, so I went with it, but calling it out in case it has bad consequences.

Reply:
In theory, publicKey is never null because it's from a QRC file. Probably it could be in UNIT_TEST.
Same for content.

Reply:
if you are worried for the unit-test, we can add a `#if defined(UNIT_TEST) ... return true..`

Reply:
Plus, I don't understand the type of comparison here. == nullptr seems to me that you want to check if a pubKey pointer is null. But this is a reference. Maybe we want to check this `isEmpty()` instead.

Reply:
> In theory, publicKey is never null because it's from a QRC file. Probably it could be in UNIT_TEST.

Maybe I can just check if signature is empty?

> if you are worried for the unit-test, we can add a #if defined(UNIT_TEST) ... return true..

That is what we have currently. Regardless of the arguments we return `true` in tests. However I needed it to return `false` in tests in case the signature was missing, otherwise my tests were failing. 

> Plus, I don't understand the type of comparison here. == nullptr seems to me that you want to check if a pubKey pointer is null. But this is a reference. Maybe we want to check this isEmpty() instead.

I'll try with isEmpty!